{{cookiecutter.project_slug}}/clients/web/vue3/src/router/index.js

```diff
@@ -15,28 +15,31 @@ const routes = [
     // this generates a separate chunk (about.[hash].js) for this route
     // which is lazy-loaded when the route is visited.
     component: () => import(/* webpackChunkName: "login" */ '../views/Login.vue'),
+    beforeEnter: requireNoAuth,
   },
```

Comment:
I think we should merge these two into one chunk rather than two they are both very small

Reply:
The chunks here are saying that each one of these files is it's own chunk. The reason we chunk them is for lazy loading in this case we are saying we should lazy load them all but really that just generates a bunch of small file chunks that don't really affect size/loading time